
<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Structural Biology - AI4Science101</title>
    <link rel="stylesheet" href="/assets/css/app.css">
    <link rel="shortcut icon" type="image/png"
           href="/favicon.png" 
    />
    <script defer src="https://use.fontawesome.com/releases/v5.3.1/js/all.js"></script>
    <!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Structural Biology | AI4Science101</title>
<meta name="generator" content="Jekyll v4.3.2" />
<meta property="og:title" content="Structural Biology" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="AI for Science 101" />
<meta property="og:description" content="AI for Science 101" />
<link rel="canonical" href="http://localhost:4000/blogs/structural_biology/" />
<meta property="og:url" content="http://localhost:4000/blogs/structural_biology/" />
<meta property="og:site_name" content="AI4Science101" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Structural Biology" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","description":"AI for Science 101","headline":"Structural Biology","url":"http://localhost:4000/blogs/structural_biology/"}</script>
<!-- End Jekyll SEO tag -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-183522810-1"></script>
<script>
  window['ga-disable-UA-183522810-1'] = window.doNotTrack === "1" || navigator.doNotTrack === "1" || navigator.doNotTrack === "yes" || navigator.msDoNotTrack === "1";
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'UA-183522810-1');
</script><!-- head scripts --></head>

  <body>
    
<nav class="navbar is-primary" >
    <div class="container">
        <div class="navbar-brand">
            <a class="navbar-brand" href="/">
            <span><img src="/logonav.png" alt="Logo" style="height: auto; width: auto; max-height: 45px; max-width: 250px;"></span>
            </a>
            <a role="button" class="navbar-burger burger" aria-label="menu" aria-expanded="false" data-target="navMenu">
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
                <span aria-hidden="true"></span>
            </a>
        </div>
        <div class="navbar-menu" id="navMenu">
            <div class="navbar-end">
                
                
                    
                    <a href="/" class="navbar-item ">Home</a>
                    
                
                    
                    <div class="navbar-item has-dropdown is-hoverable">
                        <a href="/blogs/" class="navbar-link ">Blogs</a>
                        <div class="navbar-dropdown">
                            
                            <a href="/blogs/announcement/" class="navbar-item ">Announcement</a>
                            
                            <a href="/blogs/ai_for_science_science/" class="navbar-item ">AI for Scientific Discovery</a>
                            
                            <a href="/blogs/ai_for_science_ai/" class="navbar-item ">Scientific Discovery in the era of AI</a>
                            
                            <a href="/blogs/molecular_simulation/" class="navbar-item ">Molecular Simulation</a>
                            
                            <a href="/blogs/gaussian_processes/" class="navbar-item ">Gaussian Processes</a>
                            
                            <a href="/blogs/causal_machine_learning/" class="navbar-item ">Causal Machine Learning</a>
                            
                            <a href="/blogs/statistical_learning_theory/" class="navbar-item ">Statistical Learning Theory</a>
                            
                            <a href="/blogs/optimization/" class="navbar-item ">Optimization</a>
                            
                            <a href="/blogs/graph_machine_learning/" class="navbar-item ">Graph Machine Learning</a>
                            
                            <a href="/blogs/reinforcement_learning/" class="navbar-item ">Reinforcement Learning</a>
                            
                            <a href="/blogs/quantum_science/" class="navbar-item ">Quantum Science</a>
                            
                            <a href="/blogs/comp_sustainability/" class="navbar-item ">Computational Sustainability</a>
                            
                            <a href="/blogs/structural_biology/" class="navbar-item is-active">Structural Biology</a>
                            
                            <a href="/blogs/quantum_chemistry/" class="navbar-item ">Quantum Chemistry</a>
                            
                            <a href="/blogs/knowledge_base/" class="navbar-item ">Knowledge Base</a>
                            
                        </div>
                    </div>
                    
                
                    
                    <a href="/resources/" class="navbar-item ">Resources</a>
                    
                
                
            </div>

        </div>
    </div>
</nav>

    
    


    <section class="section">
        <div class="container">
            <div class="columns">
                
                <div class="column is-4-desktop is-4-tablet">
                    

<aside class="menu">

</aside>
                </div>
                
                <div class="column is-8">
                    
                    
                    
                    
    
    

<div class="contents">
    <div class="menu">
        <p class="menu-label">Structural Biology</p>
        <ul class="menu-list">
  <li><a href="#what-is-rna">What is RNA?</a></li>
  <li><a href="#why-do-we-study-rna">Why Do We Study RNA?</a></li>
  <li><a href="#why-do-we-study-rna-with-aiml">Why Do We Study RNA with AI/ML?</a></li>
  <li><a href="#rna-structure-prediction">RNA Structure Prediction</a>
    <ul>
      <li><a href="#sec:headings">RNA Secondary Structure</a></li>
      <li><a href="#rna-tertiary-structure">RNA Tertiary Structure</a></li>
    </ul>
  </li>
  <li><a href="#interaction-of-rna-with-other-substances">Interaction of RNA with other substances</a>
    <ul>
      <li><a href="#rna-binding-protein">RNA Binding Protein</a></li>
      <li><a href="#rna-with-small-molecule">RNA with Small Molecule</a></li>
    </ul>
  </li>
  <li><a href="#rna-sequence-optimization">RNA Sequence Optimization</a></li>
  <li><a href="#rna-function-prediction">RNA Function Prediction</a>
    <ul>
      <li><a href="#protein-rna-interaction">Protein-RNA interaction</a></li>
      <li><a href="#mrna-untranslated-regions-function">mRNA untranslated region’s function</a></li>
      <li><a href="#analysis-of-single-cell-rna-sequencing-data">Analysis of single-cell RNA sequencing data</a></li>
      <li><a href="#interaction-between-lncrna-and-rna-binding-protein">Interaction between LncRNA and RNA binding protein</a></li>
    </ul>
  </li>
  <li><a href="#successful-cases">Successful Cases</a></li>
  <li><a href="#recommended-materials-for-studying-rna-with-aiml">Recommended Materials for Studying RNA with AI/ML</a></li>
  <li><a href="#what-is-protein-and-protein-structure">What is Protein and Protein Structure?</a></li>
  <li><a href="#protein-structure-prediction-1">Protein Structure Prediction</a></li>
  <li><a href="#references">References</a></li>
</ul>
    </div>
</div>




<div class="content">
    <script type="text/javascript" async="" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-MML-AM_CHTML">
</script>

<p>Author(s): Ruichu Gu, Xi Wang, Tianfan Fu.</p>

<p><strong>Before you start</strong></p>

<ul>
  <li>
    <p>This tutorial assumes that you have already learned:</p>

    <ul>
      <li>
        <p>Basic biochemistry knowledge,</p>
      </li>
      <li>
        <p>Basic Machine Learning knowledge;</p>
      </li>
    </ul>
  </li>
  <li>
    <p>Deeply explore this area you need to master the following:</p>

    <ul>
      <li>
        <p>Molecular Biology,</p>
      </li>
      <li>
        <p>Computational structural biology,</p>
      </li>
      <li>
        <p>Biochemistry,</p>
      </li>
      <li>
        <p>Machine Learning &amp; Deep Learning.</p>
      </li>
    </ul>
  </li>
</ul>

<h1 id="introduction">Introduction</h1>

<h2 id="what-is-rna">What is RNA?</h2>

<p>Ribonucleic acids (RNA) are polymeric molecules, assembled from a chain
of nucleotides. Different from DNA, however, RNA is often
single-stranded. An RNA molecule has a backbone made of alternating
phosphate groups and the sugar ribose, rather than the deoxyribose found
in DNA. Attached to each sugar is one of four bases: adenine (A), uracil
(U), cytosine (C), or guanine (G).</p>

<center><img src="/img/Structural_Biology/101_draft_01.png" /></center>
<center>Figure 1: A general framework for computational structural biology.</center>
<p><br /></p>

<p>Here we give a simple framework to sort out the related work of
computational structural biology in recent years. Based on the central
dogma, we will introduce related studies on nucleic acids and proteins
from the perspectives of structure and function, respectively. At the
same time, we will also combine commonly used research methods in the
field of structural biology, so that readers can have a general
understanding of the research process of structural biology after
reading. Readers can search for relevant information by themselves to
further understand the subject background and its work.</p>

<center><img src="/img/Structural_Biology/Ribonucleic-Acid-RNA.png" /></center>
<center>Figure 2: Basic Structure of RNA.</center>
<p><br /></p>

<p>Each nucleotide in RNA contains a ribose sugar, with carbons numbered 1’
through 5’. A base is attached to the 1’ position, in general, adenine
(A), cytosine (C), guanine (G), or uracil (U). A phosphate group is
attached to the 3’ position of one ribose and the 5’ position of the
next. The phosphate groups have a negative charge each, making RNA a
charged molecule (polyanion). The bases form hydrogen bonds between
cytosine and guanine, between adenine and uracil and between guanine and
uracil. RNA plays an important role in biological processing (Figure
3). Different types of RNA exist in cells: messenger RNA (mRNA), ribosomal RNA (rRNA)
and other non-coding RNA. Cellular organisms use messenger RNA (mRNA) to
convey information from DNA to proteins, which is so-called coding RNA.
Besides, non-coding RNA, containing no genetic information, participates
in catalyzing biological reactions, controlling gene expression, or
sensing and communicating responses to cellular signals through
specialized structures.</p>

<center><img src="/img/Structural_Biology/RNA_family.png" /></center>
<center>Figure 3: RNA Family and Function.</center>
<p><br /></p>

<h2 id="why-do-we-study-rna">Why Do We Study RNA?</h2>

<p>Among the pathogenic proteins related to human diseases, more than 80%
of the proteins cannot be targeted by the current conventional small
molecule drugs and biological macromolecular preparations, which are
non-drugable protein targets. RNA-targeted drugs treat from the
transcriptional level, which can greatly expand the proportion of
therapeutic targets in the human genome. They have the advantages of
abundant candidate targets, short research and development cycle,
long-lasting drug efficacy, and high success rate of clinical
development. They are expected to become the third wave of modern new
drugs after drugs and antibody drugs.<br />
RNA-targeted drugs are a completely new class of drugs that are
completely different from small-molecule drugs and antibody drugs. On
the one hand, they can target intracellular mRNA, ncRNA, etc., and
inhibit the expression of target proteins through gene silencing to
achieve the purpose of treating diseases; on the other hand, It is also
possible to develop a new generation of vaccines and protein replacement
therapy based on mRNA, which is expected to overcome diseases for which
there is no drug, including genetic diseases and other intractable
diseases. The future potential application scenarios of RNA-targeted
drugs are broad.<br />
After nearly 40 years of development, oligonucleotide-based RNA-targeted
drug development has made significant progress in delivery technology,
stable chemical modification, pharmaceutical research, and clinical
translation, making the clinical effect of RNA-targeted drugs gradually
get verified. In August 2020, the world’s first small molecule RNA
targeting drug Evrysdi was approved by the FDA.</p>

<h2 id="why-do-we-study-rna-with-aiml">Why Do We Study RNA with AI/ML?</h2>

<p>There are several reasons why AI and machine learning can be valuable
tools in studying RNA:</p>

<ol>
  <li>
    <p>RNA is a complex molecule: RNA molecules can have a variety of
 structures and functions, making it difficult to study them using
 traditional methods. AI and deep learning can be used to analyze
 large amounts of data and identify patterns that might not be
 noticeable through manual inspection.</p>
  </li>
  <li>
    <p>Large amount of data: RNA-seq and other high-throughput sequencing
 technologies have generated a huge amount of data, making it
 difficult to analyze manually. AI and deep learning algorithms can
 help process and make sense of this data, reducing the time and
 effort required.</p>
  </li>
  <li>
    <p>Predictive modeling: AI and deep learning can be used to make
 predictions about RNA structure and function based on sequence data.
 This can help researchers identify potential targets for drug
 development or understand the mechanisms of RNA-related diseases.</p>
  </li>
</ol>

<h1 id="current-status-of-rna-research">Current Status of RNA Research</h1>

<h2 id="rna-structure-prediction">RNA Structure Prediction</h2>

<p>The structural features of RNA are closely related to its physiological
functions. RNA secondary structure and tertiary structure prediction are
popular research directions at present.</p>

<h3 id="sec:headings">RNA Secondary Structure</h3>

<p>Secondary structure plays an important role in determining the function
of noncoding RNAs. Hence, identifying RNA secondary structures is of
great value to research. Over the years, scientists have developed a set
of tools for secondary structure prediction, where remarkable progress
has been achieved. However, the problem is still unsolved until now.
This section will give a brief introduction to RNA secondary structure
and related machine learning algorithms.</p>

<center><img src="/img/Structural_Biology/RNA-SS.png" /></center>
<center>Figure 4: RNA Secondary Structure.</center>
<p><br /></p>

<p>Similar to proteins, RNA forms secondary structures through hydrogen
bonds between bases. Stem is the most important component of RNA
secondary structure, and all other secondary structures, except
pseudo-knots, can be deduced from stems. For example, Internal Loop is a
combination of two stems. See Figure 4. For representation of the secondary structure, we
have three widely used methods.</p>

<h4 id="adjacency-matrix">Adjacency matrix</h4>

<p>. The adjacency matrix consisting of 0 and 1. \(a_{ij=1}\) denotes the
base pairing at the \(i\)th and \(j\)th positions on the sequence to form a
hydrogen bond (Figure 5). Here the adjacency matrix is a perfectly
symmetric matrix.</p>

<center><img src="/img/Structural_Biology/SS_adj.png" /></center>
<center>Figure 5: Adjacency Matrix of RNA Secondary Structure.</center>
<p><br /></p>

<h4 id="connectivity-table">Connectivity Table</h4>

<p>The first line of the CT file includes the total number of nucleotides,
the folding energy, and the file name. In the other rows, each row
contains 6 columns of data, which are: i) List of the nucleotides from 1
to N (N = total number of nucleotides); ii) List of the type of
nucleotide (A, G, U, or C); iii) List of the nucleotides increasing from
zero to N - 1; iv) List of the nucleotides from 2 to N; v) List of the
nucleotides that are paired to those listed in increasing order. Any
zeros in the fifth column indicate that the particular nucleotide is
unpaired; vi) A repeat of column i.</p>

<center><img src="/img/Structural_Biology/ct_SS.png" /></center>
<center>Figure 6: Connectivity Table of RNA Secondary Structure.</center>
<p><br /></p>

<h4 id="dot-bracket-annotation">Dot Bracket Annotation</h4>

<p>The unpaired bases in the RNA sequence are denoted by ".", and the two
bases that form hydrogen pairs are denoted by "(" and ")". Bases in
the base pair relatively close to the 5’ end are denoted by "(", and
bases in the base pair relatively close to the 3’ end are denoted by
")".</p>

<center><img src="/img/Structural_Biology/db_SS.png" /></center>
<center>Figure 7: Dot Bracket Annotation of RNA Secondary Structure.</center>
<p><br /></p>

<p>For researchers, RNA secondary structure prediction can be viewed as
multi-classification or pixel prediction problems. In recent years,
there have been lots of excellent work that focus on these challenges.
Even though more mature RNA secondary structure datasets are now
available, their total sample size is still relatively small. Commonly
used datasets and their URLs are listed in Table 1.</p>
<div style="text-align: center;">
Table 1: RNA Secondary Structure Database
  <table>
    <thead>
      <tr>
        <th>Database</th>
        <th>URLs</th>
      </tr>
    </thead>
    <tbody>
      <tr>
        <td>bpRNA</td>
        <td><a href="http://bprna.cgrb.oregonstate.edu">bpRNA Website</a></td>
      </tr>
      <tr>
        <td>RNAStralign [2]</td>
        <td><a href="https://github.com/mehdi1902/RNA-secondary-structure-prediction-using-CNN">RNAStralign GitHub</a></td>
      </tr>
      <tr>
        <td>ArchiveII [3]</td>
        <td><a href="https://github.com/mehdi1902/RNA-secondary-structure-prediction-using-CNN">ArchiveII GitHub</a></td>
      </tr>
    </tbody>
  </table>
</div>

<p>We already have lots of strategies for RNA representation. The simplest
one is using one hot encoding. For ‘A’, ‘U’, ‘C’, and ‘G’ bases, four
dimensions coding is enough. Besides, under IUPAC standard, more codes
are used (Table 2).</p>

<div style="text-align: center;">
  Table 2: IUPAC Standard Codes for DNA/RNA
  <table>
    <thead>
      <tr>
        <th>IUPAC nucleotide code</th>
        <th>Base</th>
      </tr>
    </thead>
    <tbody>
      <tr><td>A</td><td>Adenine</td></tr>
      <tr><td>C</td><td>Cytosine</td></tr>
      <tr><td>G</td><td>Guanine</td></tr>
      <tr><td>T (or U)</td><td>Thymine (or Uracil)</td></tr>
      <tr><td>R</td><td>A or G</td></tr>
      <tr><td>Y</td><td>C or T</td></tr>
      <tr><td>S</td><td>G or C</td></tr>
      <tr><td>W</td><td>A or T</td></tr>
      <tr><td>K</td><td>G or T</td></tr>
      <tr><td>M</td><td>A or C</td></tr>
      <tr><td>B</td><td>C or G or T</td></tr>
      <tr><td>D</td><td>A or G or T</td></tr>
      <tr><td>H</td><td>A or C or T</td></tr>
      <tr><td>V</td><td>A or C or G</td></tr>
      <tr><td>N</td><td>any base</td></tr>
      <tr><td>. or -</td><td>gap</td></tr>
    </tbody>
  </table>
</div>

<p>Besides one hot encoding, there are other methods of representations.
Multiple Sequence Alignment (MSA) is a vital method for genetic study,
which contain co-evolutionary information. The AlphaFold2 and other
models have proved the power of MSA for protein structure prediction.
For RNA, we can also use MSA as model inputs. 2D features generated from
MSA are also useful, for examples, Position-specific scoring matrix
(PSSM), GREMLIN Matrix, and plmDCA. Some probability matrix calculated
by free-energy based algorithms can also be used as input features, for
instance, LinearFoldPartition provides rapid and accurate calculation of
bases-pair probability matrix. Very recently, Li. et al. reported a
large pre-trained models, which gives sequence embedding with 640
dimensions [4]. Latent embedding from pre-trained
models are also powerful. Commonly-used features for RNA are listed in
Table 3.</p>

<!-- <div align="center"> -->

<center>Table 3: Features for RNA</center>

<table>
  <thead>
    <tr>
      <th>Features</th>
      <th>Descriptions</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>One-hot Encoding</td>
      <td>Discrete feature encoding in Euclidean space</td>
    </tr>
    <tr>
      <td>Probability matrix</td>
      <td>Base pairing probability matrix calculated based on thermodynamic principle</td>
    </tr>
    <tr>
      <td>MSA</td>
      <td>Multi-species sequence similarity relationships</td>
    </tr>
    <tr>
      <td>PSSM</td>
      <td>Sequence position-specific base frequencies</td>
    </tr>
    <tr>
      <td>GREMLIN</td>
      <td>Markov random field-based graph model</td>
    </tr>
    <tr>
      <td>Pre-trained Embedding</td>
      <td>Latent sequences embedding</td>
    </tr>
  </tbody>
</table>

<!-- </div> -->

<h4 id="1d-approaches">1D Approaches</h4>

<p>The 1D approaches treat RNA secondary structure as sequences. For
instance, by using dot-bracket annotation as target, researchers convert
problems into multi-classification tasks. The ".", "(", and ")"
can be viewed as three classes. This is a traditional "seq2seq"
problems. Researchers can use relevant models from the field of Natural
Language Processing (NLP) to solve this problem, such as RNN, LSTM, and
Transformer. 1D approaches use a loss function similar to that of
classification problems, such as cross-entropy. Class imbalance may
occur for such classification problems due to the fact that the number
of paired bases on an RNA sequence is often less than the number of
unpaired bases.</p>

<h4 id="2d-approaches">2D Approaches</h4>

<p>The 2D approach focuses on the adjacency matrix as the prediction
target, which is an \(L*L\) two-dimensional matrix, where each element
value represents the base pairing probability. The 2D approach for RNA
secondary structure prediction has a strong similarity to the semantic
segmentation task in computer vision. Researchers can use similar ideas
to study RNA secondary structure prediction problems. For example,
Ufold [5] used the classical Unet model in semantic
segmentation and achieved excellent results even with simple features.</p>

<h3 id="rna-tertiary-structure">RNA Tertiary Structure</h3>

<p>The tertiary structure prediction of RNA is similar to that of proteins,
again trying to obtain the 3D coordinates of the atoms. However, the
tertiary structure prediction of RNA is still dominated by molecular
simulation methods, and machine learning methods are still in a
relatively rudimentary stage, mainly due to the lack of a large amount
of high-precision tertiary structure data at present. Even though
molecular simulation methods are well developed, they still consume long
computational time and are moderately accurate. It is worth noting that
molecular simulation methods are able to obtain tertiary structures with
high accuracy if high-precision secondary structures are used as
constraints. In addition, borrowing ideas from earlier protein structure
prediction, using machine learning methods to obtain contact maps as
fuzzy inputs to molecular simulation algorithms is also a feasible
approach. In addition, Li and colleagues proposed an end-to-end
structure prediction model E2Efold-3D.
Table 4
lists some machine learning methods for RNA tertiary structure
prediction.</p>

<center>Table 4: RNA Tertiary Structure Prediction</center>

<table>
  <thead>
    <tr>
      <th>Models</th>
      <th>Authors</th>
      <th>Date</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Euclidean distance neural networks [1]</td>
      <td>Nikolay V. Dokholyan</td>
      <td>2022</td>
    </tr>
    <tr>
      <td>Geometric Potentials from Deep Learning [2]</td>
      <td>Yang Zhang</td>
      <td>2022</td>
    </tr>
    <tr>
      <td>E2Efold-3D: End-to-End Deep Learning Method [3]</td>
      <td>Yu Li</td>
      <td>2022</td>
    </tr>
  </tbody>
</table>

<h2 id="interaction-of-rna-with-other-substances">Interaction of RNA with other substances</h2>

<h3 id="rna-binding-protein">RNA Binding Protein</h3>

<p>RNA Binding proteins are key regulators of cellular and developmental
processes and play a key role in post-transcriptional splicing and
localization of mRNA. Defining the properties of RBP will allow us to
regulate the protein better. RBP is usually combined with specific
motifs in RNA structures through RNA Binding domains.<br />
At present, the most definitive way to study RNA binding to proteins is
to extract them from high-resolution experimental structures of
protein-RNA complexes, which is expensive and time-consuming. Therefore,
it is of great significance to develop available computational methods
to identify Binding information in RNA Binding proteins.</p>

<h3 id="rna-with-small-molecule">RNA with Small Molecule</h3>

<p>The increasing appreciation for the crucial roles of RNAs in infectious
and non-infectious human diseases makes them attractive therapeutic
targets. Coding and non-coding RNAs frequently fold into complex
conformations which, if effectively targeted, offer opportunities to
therapeutically modulate numerous cellular processes, including those
linked to undruggable protein targets. RNA is a legitimate target of
small molecules and that ligands can indeed bind discrete binding
pockets in RNA, cementing the applicability of small-molecule treatment
to target RNAs.<br />
Overall, three major components are required to support the development
of efficacious RNA-based small molecule therapeutics: (1) the
identification of therapeutically valuable RNA target, (2) the
development of a screening method that will establish drug-like
molecules against target RNA, and (3) the identification of RNA motifs
that accommodate ligand binding with high affinity and specificity.
Current ML techniques can be used to screen small molecules and evaluate
the binding specificity of small molecules, which can improve the
selection of small molecule structures with RNA affinity from a large
number of unknown targets.</p>

<h2 id="rna-sequence-optimization">RNA Sequence Optimization</h2>

<p>The epidemic of COVID-19 has promoted the research and development of
RNA vaccines. In addition to the encapsulation and delivery system, the
selection of appropriate RNA sequences can enable the vaccine to be
expressed in host cells more efficiently. At present, the relevant
optimization studies mainly focus on prokaryotes. Due to the complex
transcription-translation mechanism of eukaryotes, there is no good
optimization method.</p>

<h2 id="rna-function-prediction">RNA Function Prediction</h2>

<p>RNA function prediction refers to the task of exploring the biological
function of RNA. In particular, non-coding RNAs, which play regulatory
and catalytic roles in cells by binding to other macromolecules such as
proteins, as well as RNAs that are beacons for certain diseases, are
important research topics in medical research. In this article, we list
some common RNA function prediction tasks for which machine learning
techniques may bring revolutionary changes.</p>

<h3 id="protein-rna-interaction">Protein-RNA interaction</h3>

<p>RNA-protein interactions is crucial to discovery the mechanisms and
control of gene expression and biological synthesis. Traditional
molecular simulation methods may suffer from low accuracy and long
computing time. With the development of machine learning technology in
recent years, this classical field may welcome revolution.</p>

<h3 id="mrna-untranslated-regions-function">mRNA untranslated region’s function</h3>

<p>Untranslated Region (UTR) refers to the region on the mRNA that does not
carry genetic information, which is usually at both ends of the Coding
Sequence (CDS) region. Biologists have found that specific sites on it
play an important role in gene expression and regulation. Therefore, the
delicate identification of the UTR region can help biologists to
discover new regulatory sites for pathogenesis and drug development.</p>

<h3 id="analysis-of-single-cell-rna-sequencing-data">Analysis of single-cell RNA sequencing data</h3>

<p>Single-cell RNA sequencing (scRNA-seq) refers to profile the whole
transcriptome of a large number of individual cells. scRNA-seq helps
biologists to uncover gene expression information of single cells more
precisely, i.e. to identify expression differences between different
cells in the same tissue. However, due to the huge amount of data it
generates, data processing is a more difficult step in scRNA-seq.
Machine learning methods are good at extracting knowledge from huge
amount of information, so applying ML in this field is a promising
approach.</p>

<h3 id="interaction-between-lncrna-and-rna-binding-protein">Interaction between LncRNA and RNA binding protein</h3>

<p>Up to now, the majority of Long non-coding RNA (LncRNA) functions are
mediated by lncRNA-protein interactions. lncRNAs perform diverse
functions by forming RNA-protein complexes with proteins, such as
chromosomal regulatory complexes, transcription factors and
ribonucleoprotein (RNP) complexes. Therefore, techniques to study
LncRNA-protein interactions can help us unravel the biological functions
of LncRNAs.</p>

<h1 id="outlook-ai-and-rna-study">Outlook: AI and RNA Study</h1>

<p>Problems are getting harder for scientists to solve due to the enormity
of the data that needs processing in the big data era. It’s tough for
researchers to catch the ‘Big Pictures’. The treasure of RNA and
RNA-related drugs are hidden in the intricate books of genes and cells.
Artificial Intelligence (AI) technique is expert at exploring that high
dimensional puzzles. It looks promising that applying AI in RNA
sciences. Here we will give two successful examples for AI and RNA.</p>

<h2 id="successful-cases">Successful Cases</h2>

<ul>
  <li>ARES</li>
</ul>

<p>In 2021, Stanford University researcher reported a new deep-learning
system called Atomic Rotationally Equivariant Scorer (ARES)
significantly improves the prediction of RNA structures over previous
artificial intelligence (AI) models. Although RNA molecules fold into
intricate 3D structures like proteins that allow them to perform a wide
range of cellular functions, there is far less training data available
for RNA structures prediction. To address this problem, ARES accept
simple 3D coordinates and chemical element type of each atom as input
with less preprogrammed assumptions. With FARFAR2, ARES achieved
excellent performance on RNA puzzles dataset. The clever design of the
neural network guaranteed generalizing ability. Besides, start from
physical view is also critical. Learning from the essential properties
of the sturctures makes it convincing and reasonable.</p>

<ul>
  <li>RNA-Binding Protein</li>
</ul>

<p>Interactions with RNA-binding proteins (RBPs) are integral to RNA
function and cellular regulation, and dynamically reflect specific
cellular conditions. However, current methods can’t predict dynamic RBP
binding in various cellular conditions. By using in vivo data and deep
learning models, Researchers from Tsinghua University developed a power
AI tools for accurate prediction of RBP binding problems. The rich
profiling data and deep learning-based prediction tool provide access to
a previously inaccessible layer of cell-type-specific RBP–RNA
interactions, with clear utility for understanding and treating human
diseases. One of the most obstacles of RBP would be how to extract rules
and knowledge from millions to billions of samples. Fortunately, handing
large-scale data is the speciality of AI/ML. It’s promising the AI/ML
will bring breakthrough progress.</p>

<h2 id="recommended-materials-for-studying-rna-with-aiml">Recommended Materials for Studying RNA with AI/ML</h2>

<center>Table 5: Reading Materials</center>

<table>
  <thead>
    <tr>
      <th>Titles</th>
      <th>Authors</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Artificial Intelligence and Molecular Biology</td>
      <td>Lawrence E. Hunter</td>
    </tr>
    <tr>
      <td>Molecular Biology of RNA</td>
      <td>David Elliott, Michael Ladomery</td>
    </tr>
    <tr>
      <td>Deep Learning</td>
      <td>Ian Goodfellow, Yoshua Bengio, Aaron Courville</td>
    </tr>
  </tbody>
</table>

<h1 id="protein-structure-prediction">Protein Structure Prediction</h1>

<p>The discovery of targets is greatly impacted by the three-dimensional
structures of proteins, given that a protein’s biological role
predominantly arises from its spatial configuration. Nobel laureate Dr.
Christian Anfinsen proposed in 1972 that the 3D structure of a protein
should be definitively determined by its sequence of amino
acids [9]. In most instances, researchers initiate
their work with knowledge of a protein’s amino acid sequence, which is
derived from genetic information. Subsequently, they strive to ascertain
its three-dimensional structure to gain deeper insights into its
physical, chemical, and biological attributes. Conventional experimental
techniques, such as X-ray crystallography [10], can precisely
elucidate protein structures, yet these methodologies are
resource-intensive and time-demanding. Out of the approximately 1
billion documented protein amino acid sequences, only around 100
thousand structures have been identified using these approaches,
encompassing approximately 17% of human
proteins [11].</p>

<p>Protein Structure Prediction, also known as protein folding, aims at
predicting the 3D structure of protein given the amino acid sequence.
This hypothesis ignited a fifty-year-long endeavor to predict protein 3D
structures computationally using their amino acid sequence as a
foundation.</p>

<h2 id="what-is-protein-and-protein-structure">What is Protein and Protein Structure?</h2>

<p>Proteins are the building blocks of life. Every cell in the human body
contains proteins. Proteins carry out all the essential functions in the
human body. Proteins support almost all biological activities. For
example, stomach enzymes serve to digest food; the movement of
myosin on actin is the driving force for muscular contraction; whey
proteins strengthen cell anti-oxidation; and antibodies protect us from
disease.</p>

<p>The basic protein structure is a chain of amino acids. The amino acid
chain can fold into a 3D structure.</p>

<p>Four levels of hierarchical structures usually represent a protein: (1)
primary structure (i.e., amino acid sequence), (2) secondary structure,
(3) tertiary structure, and (4) quaternary structure. They characterize
the hierarchical structure of a protein at different levels of
complexity, as described below.</p>

<ol>
  <li>
    <p><strong>Primary structure</strong>. The primary structure of a protein is simply
 the sequence of amino acids in a polypeptide chain. There are a
 total of 20 natural amino acids, as shown in
 Table 6.</p>
  </li>
  <li>
    <p><strong>Secondary structure</strong>. The secondary structure of a protein refers
 to regular, recurring arrangements in the space of adjacent amino
 acid residues in a polypeptide chain. It characterizes the localized
 conformation of the polypeptide chain. There are a total of 8
 categories of secondary structures, e.g., \(\alpha\)-helix, Turn,
 Bend, \(\pi\)-helix, 3-10 helix, Strand, Isolated beta-bridge residue,
 and None (also called a coil, which is not in any above formations).
 Each amino acid belongs to one of these eight secondary structures.</p>
  </li>
  <li>
    <p><strong>Tertiary structure</strong>. The tertiary structure of a protein refers
 to the overall three-dimensional arrangement of its polypeptide
 chain in Euclidean space. An amino acid consists of an \(\alpha\)
 (central) carbon atom linked to an amino group, a carboxyl group, a
 hydrogen atom, and a variable component called a side chain. 3D
 graph structure is also known as the backbone structure, which
 belongs to the tertiary structure of a protein.</p>
  </li>
  <li>
    <p><strong>Quaternary structure</strong>. The quaternary structure of a protein
 describes the relationship of multiple polypeptide chains (or
 subunits) into a closely packed arrangement. Each polypeptide chain
 has its own primary, secondary, and tertiary structure. Hydrogen
 bonds and van der Waals forces between nonpolar side chains arrange
 the subunits together. It lies in the fourth and highest level of
 protein structure. The proteins with a single polypeptide chain do
 not have quaternary structures.</p>
  </li>
</ol>

<h2 id="protein-structure-prediction-1">Protein Structure Prediction</h2>

<p>We discuss some essential components for protein structure prediction.</p>

<ul>
  <li>
    <p><strong>Feature engineering</strong>.</p>

    <ol>
      <li>
        <p><strong>Multiple sequence alignment (MSA)</strong> is a feature engineering
 technique to align three or more biological sequences (generally
 protein or DNA) and compare them for similarity. It is an
 essential tool in bioinformatics and is used in various
 applications, including phylogenetic tree construction, protein
 domain identification, protein structure and function
 prediction, and understanding evolutionary processes. MSA is
 used for most protein folding methods, including
 RaptorX-Contact [12],
 AlphaFold1 [13],
 trRoseTTA [14], AlphaFold2 [15],
 RoseTTAFold [16],
 AlphaFold-Multimer [17], UniFold [18],
 and OpenFold [19].</p>
      </li>
      <li>
        <p><strong>Language model (LM)</strong>. The process of Multiple Sequence
 Alignment (MSA) is often characterized by its time-intensive and
 resource-heavy nature, primarily due to its computationally
 intensive approach. To address this challenge,
 ESMFold [20] offers a solution by employing a
 large-scale language model that is pre-trained on amino acid
 sequences. This approach replaces the conventional MSA with a
 robust neural representation, substantially accelerating the
 process.</p>
      </li>
    </ol>
  </li>
  <li>
    <p><strong>Learning strategy.</strong> These are two kinds of learning strategies
  for the existing protein folding methods.</p>

    <ol>
      <li>
        <p><strong>Two-stage learning.</strong> The earlier protein folding approaches
 leverage two-stage learning: (1) predicts pairwise distance and
 torsion angles and (2) constructs potential function based on
 the predicted distance and torsion angle and optimizes it with
 gradient descent. The strategy is used in
 RaptorX-Contact [12],
 AlphaFold1 [13],
 trRoseTTA [14].</p>
      </li>
      <li>
        <p><strong>End-to-end learning.</strong> End-to-end learning is a powerful
 learning approach that utilizes gradient-based optimization on
 the entire system, which can comprise various differentiable
 components such as neural network modules. This training
 strategy allows the system to learn rich representations
 directly from raw inputs, eliminating the need for manual
 feature engineering. The strategy is used in
 AlphaFold2 [15], RoseTTAFold [16],
 AlphaFold-Multimer [17], UniFold [18],
 and OpenFold [19].</p>
      </li>
    </ol>
  </li>
  <li>
    <p><strong>Neural Network.</strong></p>

    <ol>
      <li>
        <p><strong>Residual Convolutional Neural Network</strong>. Residual CNN takes
 the MSA feature as input. The groundtruth of residual CNN is the
 distance matrix and torsion angle, which are translation- and
 rotation-invariant. Thus, it is \(SE(3)\)-invariant. It uses
 dilated Convolutional Neural Network, a variant of convolutional
 neural networks that incorporates dilated convolutions. Dilated
 convolutions introduce a dilation rate parameter that controls
 the spacing between sampled elements within the convolutional
 kernel, affecting the receptive field size of the output; e.g.,
 a dilation rate of 2 means sampling every two elements. The main
 feature of dilated convolutions is the ability to increase the
 network’s receptive field, meaning each output unit can "see"
 a wider region of the input. Residual connection is used to
 alleviate gradient vanishment issues. Residual CNN is used in
 RaptorX-Contact [12],
 AlphaFold1 [13],
 trRoseTTA [14].</p>
      </li>
      <li>
        <p><strong>Evoformer</strong>. The Evoformer is a variant of the transformer
 architecture specifically designed to leverage evolutionary
 sequence information. It stacks 48 basic transformer blocks
 repeatedly with residual connection to alleviate gradient
 vanishment issues. There is no parameter sharing between these
 blocks. For each block, the input and output are of the same
 shapes. It is \(SE(3)\)-equivariant and used in
 AlphaFold2 [15],
 AlphaFold-Multimer [17], UniFold [18],
 ESMFold [20], and
 OpenFold [19].</p>
      </li>
    </ol>
  </li>
</ul>

<p>Table 7 summarizes the different strategies for different SOTA protein folding methods. More information can be found at [21].</p>

<center>Table 6: All the 20 natural amino acids, and their abbreviations and frequencies. The frequencies are evaluated on all the processed protein data in RCSB Protein Data Bank (RCSB PDB) [RCSB PDB](https://www.rcsb.org/).</center>

<table>
  <thead>
    <tr>
      <th>Class</th>
      <th>Abbreviation</th>
      <th>Percentage (%)</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Glycine</td>
      <td>Gly / G</td>
      <td>7.6%</td>
    </tr>
    <tr>
      <td>Alanine</td>
      <td>Ala / A</td>
      <td>7.7%</td>
    </tr>
    <tr>
      <td>Valine</td>
      <td>Val / V</td>
      <td>7.0%</td>
    </tr>
    <tr>
      <td>Leucine</td>
      <td>Leu / L</td>
      <td>8.6%</td>
    </tr>
    <tr>
      <td>Isoleucine</td>
      <td>Ile / I</td>
      <td>5.5%</td>
    </tr>
    <tr>
      <td>Proline</td>
      <td>Pro / P</td>
      <td>4.6%</td>
    </tr>
    <tr>
      <td>Phenylalanine</td>
      <td>Phe / F</td>
      <td>3.6%</td>
    </tr>
    <tr>
      <td>Tyrosine</td>
      <td>Tyr / Y</td>
      <td>3.1%</td>
    </tr>
    <tr>
      <td>Tryptophan</td>
      <td>Trp / W</td>
      <td>1.2%</td>
    </tr>
    <tr>
      <td>Serine</td>
      <td>Ser / S</td>
      <td>6.7%</td>
    </tr>
    <tr>
      <td>Threonine</td>
      <td>Thr / T</td>
      <td>5.7%</td>
    </tr>
    <tr>
      <td>Cystine</td>
      <td>Cys / C</td>
      <td>1.3%</td>
    </tr>
    <tr>
      <td>Methionine</td>
      <td>Met / M</td>
      <td>2.7%</td>
    </tr>
    <tr>
      <td>Asparagine</td>
      <td>Asn / N</td>
      <td>4.1%</td>
    </tr>
    <tr>
      <td>Glutamine</td>
      <td>Gln / Q</td>
      <td>3.9%</td>
    </tr>
    <tr>
      <td>Aspartic acid</td>
      <td>Asp / D</td>
      <td>5.3%</td>
    </tr>
    <tr>
      <td>Glutamic acid</td>
      <td>Glu / E</td>
      <td>6.5%</td>
    </tr>
    <tr>
      <td>Lysine</td>
      <td>Lys / K</td>
      <td>6.3%</td>
    </tr>
    <tr>
      <td>Arginine</td>
      <td>Arg / R</td>
      <td>5.3%</td>
    </tr>
    <tr>
      <td>Histidine</td>
      <td>His / H</td>
      <td>3.1%</td>
    </tr>
  </tbody>
</table>

<center>Table 7: Summary of existing protein folding approaches</center>

<p>This table summarizes existing protein folding approaches, including 
RaptorX-Contact [12], 
AlphaFold1 [13], 
trRoseTTA [14], 
AlphaFold2 [15], 
RoseTTAFold [16], 
AlphaFold-Multimer [17], 
UniFold [18], 
ESMFold [20], 
and OpenFold [19]. 
Two-stage learning typically consists of (1) prediction of pairwise distance and orientation and (2) energy minimization. The table is copied from Artificial Intelligence in Protein Folding [21].</p>

<table>
  <thead>
    <tr>
      <th>Methods</th>
      <th>Feature</th>
      <th>Learning</th>
      <th>Network</th>
      <th>Symmetry</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>RaptorX-Contact</td>
      <td>MSA</td>
      <td>Two-Stage</td>
      <td>Residual CNN</td>
      <td>(SE(3))-Invariant</td>
    </tr>
    <tr>
      <td>AlphaFold1</td>
      <td>MSA</td>
      <td>Two-Stage</td>
      <td>Residual CNN</td>
      <td>(SE(3))-Invariant</td>
    </tr>
    <tr>
      <td>trRoseTTA</td>
      <td>MSA</td>
      <td>Two-Stage</td>
      <td>Residual CNN</td>
      <td>(SE(3))-Invariant</td>
    </tr>
    <tr>
      <td>AlphaFold2</td>
      <td>MSA</td>
      <td>End-To-End</td>
      <td>Evoformer + Structure</td>
      <td>(SE(3))-Equivariant</td>
    </tr>
    <tr>
      <td>RoseTTAFold</td>
      <td>MSA</td>
      <td>End-To-End</td>
      <td>Three-Track NN</td>
      <td>(SE(3))-Equivariant</td>
    </tr>
    <tr>
      <td>UniFold</td>
      <td>MSA</td>
      <td>End-To-End</td>
      <td>Evoformer + Structure</td>
      <td>(SE(3))-Equivariant</td>
    </tr>
    <tr>
      <td>ESMFold</td>
      <td>LM</td>
      <td>End-To-End</td>
      <td>Evoformer + Structure</td>
      <td>(SE(3))-Equivariant</td>
    </tr>
    <tr>
      <td>OpenFold</td>
      <td>MSA</td>
      <td>End-To-End</td>
      <td>Evoformer + Structure</td>
      <td>(SE(3))-Equivariant</td>
    </tr>
    <tr>
      <td>AlphaFold-Multimer</td>
      <td>MSA</td>
      <td>End-To-End</td>
      <td>Evoformer + Structure</td>
      <td>(SE(3))-Equivariant</td>
    </tr>
  </tbody>
</table>

<h2 id="references">References</h2>

<p>[1] Xinshi Chen, Yu Li, Ramzan Umarov, Xin Gao, and Le Song. “RNA secondary structure prediction by learning unrolled algorithms.” arXiv preprint arXiv:2002.05810, 2020.</p>

<p>[2] Zhen Tan, Yinghan Fu, Gaurav Sharma, and David H Mathews. “Turbofold ii: RNA structural alignment and secondary structure prediction informed by multiple homologs.” Nucleic acids research, 45(20):11570–11581, 2017.</p>

<p>[3] Michael F Sloma and David H Mathews. “Exact calculation of loop formation probability identifies folding motifs in RNA secondary structures.” RNA, 22(12):1808–1818, 2016.</p>

<p>[4] Jiayang Chen, Zhihang Hu, Siqi Sun, Qingxiong Tan, Yixuan Wang, Qinze Yu, Licheng Zong, Liang Hong, Jin Xiao, Tao Shen, et al. “Interpretable RNA foundation model from unannotated data for highly accurate RNA structure and function predictions.” bioRxiv, 2022.</p>

<p>[5] Laiyi Fu, Yingxin Cao, Jie Wu, Qinke Peng, Qing Nie, and Xiaohui Xie. “Ufold: fast and accurate RNA secondary structure prediction with deep learning.” Nucleic acids research, 50(3):e14–e14, 2022.</p>

<p>[6] Congzhou Mike Sha, Jian Wang, and Nikolay V Dokholyan. “Predicting 3D RNA structure from solely the nucleotide sequence using euclidean distance neural networks.” bioRxiv, 2022.</p>

<p>[7] Robin Pearce, Gilbert S Omenn, and Yang Zhang. “De novo RNA tertiary structure prediction at atomic resolution using geometric potentials from deep learning.” bioRxiv, 2022.</p>

<p>[8] Tao Shen, Zhihang Hu, Zhangzhi Peng, Jiayang Chen, Peng Xiong, Liang Hong, Liangzhen Zheng, Yixuan Wang, Irwin King, Sheng Wang, et al. “E2efold-3d: End-to-end deep learning method for accurate de novo RNA 3D structure prediction.” arXiv preprint arXiv:2207.01586, 2022.</p>

<p>[9] John C Wooley and Yuzhen Ye. “A historical perspective and overview of protein structure prediction.” Computational Methods for Protein Structure Prediction and Modeling: Volume 1: Basic Characterization, pages 1–43, 2007.</p>

<p>[10] MS Smyth and JHJ Martin. “X ray crystallography.” Molecular Pathology, 53(1):8, 2000.</p>

<p>[11] Kathryn Tunyasuvunakool, Jonas Adler, Zachary Wu, Tim Green, Michal Zielinski, Augustin Žídek, Alex Bridgland, Andrew Cowie, Clemens Meyer, Agata Laydon, et al. “Highly accurate protein structure prediction for the human proteome.” Nature, 596(7873):590–596, 2021.</p>

<p>[12] Sheng Wang, Siqi Sun, Zhen Li, Renyu Zhang, and Jinbo Xu. “Accurate de novo prediction of protein contact map by ultra-deep learning model.” PLOS Computational Biology, 13(1):e1005324, 2017.</p>

<p>[13] Andrew W. Senior, Richard Evans, John Jumper, James Kirkpatrick, Laurent Sifre, et al. “Improved protein structure prediction using potentials from deep learning.” Nature, 577:706–710, 2020.</p>

<p>[14] Zongyang Du, Hong Su, Wenkai Wang, Lisha Ye, Hong Wei, Zhenling Peng, Ivan Anishchenko, David Baker, and Jianyi Yang. “The trrosetta server for fast and accurate protein structure prediction.” Nature Protocols, 16(12):5634–5651, 2021.</p>

<p>[15] John Jumper, Richard Evans, Alexander Pritzel, Tim Green, Michael Figurnov, Olaf Ronneberger, Kathryn Tunyasuvunakool, Russ Bates, Augustin Žídek, Anna Potapenko, et al. “Highly accurate protein structure prediction with AlphaFold.” Nature, 596(7873):583–589, 2021.</p>

<p>[16] Minkyung Baek, Frank DiMaio, Ivan Anishchenko, Justas Dauparas, Sergey Ovchinnikov, Gyu Rie Lee, Jue Wang, Qian Cong, Lisa N Kinch, R Dustin Schaeffer, et al. “Accurate prediction of protein structures and interactions using a three-track neural network.” Science, 373(6557):871–876, 2021.</p>

<p>[17] Richard Evans, Michael O’Neill, Alexander Pritzel, Natasha Antropova, Andrew Senior, Tim Green, Augustin Žídek, Russ Bates, Sam Blackwell, Jason Yim, et al. “Protein complex prediction with alphafold-multimer.” bioRxiv, pages 2021–10, 2021.</p>

<p>[18] Ziyao Li, Xuyang Liu, Weijie Chen, Fan Shen, Hangrui Bi, Guolin Ke, and Linfeng Zhang. “Uni-fold: an open-source platform for developing protein folding models beyond alphafold.” bioRxiv, pages 2022–08, 2022.</p>

<p>[19] Gustaf Ahdritz, Nazim Bouatta, Sachin Kadyan, Qinghui Xia, William Gerecke, Timothy J O’Donnell, Daniel Berenberg, Ian Fisk, Niccolò Zanichelli, Bo Zhang, et al. “OpenFold: Retraining alphafold2 yields new insights into its learning mechanisms and capacity for generalization.” bioRxiv, pages 2022–11, 2022.</p>

<p>[20] Zeming Lin, Halil Akin, Roshan Rao, Brian Hie, Zhongkai Zhu, Wenting Lu, Nikita Smetanin, Robert Verkuil, Ori Kabeli, Yaniv Shmueli, et al. “Evolutionary-scale prediction of atomic-level protein structure with a language model.” Science, 379(6637):1123–1130, 2023.</p>

<p>[21] Xuan Zhang, Limei Wang, Jacob Helwig, Youzhi Luo, Cong Fu, Yaochen Xie, Meng Liu, Yuchao Lin, Zhao Xu, Keqiang Yan, et al. “Artificial intelligence for science in quantum, atomistic, and continuum systems.” arXiv preprint arXiv:2307.08423, 2023.</p>

</div>
                </div>
                
            </div>
        </div>
    </section>
    
        <!-- <footer class="footer">
    <div class="container">
        
        <div class="columns is-mobile">
            <div class="column is-8 has-text-left is-vcentered">
                <a class="navbar-brand" href="/">
                    <span><img src="/tdc_horizontal.png" alt="Logo" style="max-height: 40px; max-width: 250px;"></span>
                </a>
            </div>
            <div class="column is-4 has-text-right is-vcentered">
                <a href="https://arxiv.org/abs/2102.09548">
                    <span class="icon is-large">
                      <i class="fas fa-file-alt fa-3x"></i>
                    </span>
                </a>

                <a href="https://github.com/mims-harvard/TDC">
                    <span class="icon is-large">
                      <i class="fas fab fa-github fa-3x"></i>
                    </span>
                </a>

                <a href="https://twitter.com/ProjectTDC">
                    <span class="icon is-large">
                      <i class="fas fab fa-twitter fa-3x"></i>
                    </span>
                </a>

                <a href="https://join.slack.com/t/pytdc/shared_invite/zt-x0ujg5v6-zwtQZt83fhRdgrYjXRFz5g">
                    <span class="icon is-large">
                      <i class="fas fab fa-slack fa-3x"></i>
                    </span>
                </a>
            </div>
        </div>
        
    </div>
</footer> -->
    
    <script src="/assets/js/app.js" type="text/javascript"></script><!-- footer scripts --></body>
</html>

